Docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8.4-slim
+ENV PYTHONIOENCODING UTF-8
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Is it reasonable for there to be any other choice here? If we're not running any apt commands in the dockerfile, do we even need this at all?

Reply:
Maybe the expectation is that users will use this as a `FROM` image?

Reply:
> Maybe the expectation is that users will use this as a FROM image?

Yes, like this nginx image from dockerhub
https://hub.docker.com/_/nginx

Reply:
7bf1f8b

`ARG DEBIAN_FRONTEND=noninteractive` has been added.